README.md

```diff
@@ -59,6 +59,100 @@ Use the ``Deploy to Bluemix`` button **OR** create the services and run locally.
     * wbc-natural-language-understanding-service
     * wbc-tone-analyzer-service
 
+## Deploy to Kubernetes on Bluemix
+
+The following steps outline the process for deploying this application as-is to Kubernetes on IBM Bluemix.  You can read more about [Kubernetes Support and the IBM Cloud Developer Tools CLI on the Bluemix blog](https://www.ibm.com/blogs/bluemix/2017/09/deploying-kubernetes-ibm-cloud-ibm-cloud-developer-tools-cli/).
+
+1. [Create a Kubernetes cluster on Bluemix](https://console.bluemix.net/containers-kubernetes/launch?) if you do not already have one. 
+
+   *Note: This may take up to 15 minutes for a cluster to be provisioned.*
+
+2. Install the [IBM Cloud Developer Tools CLI](https://github.com/IBM-Bluemix/ibm-cloud-developer-tools/)
+
+    If you're on a MacOS or Linux machine, open up a terminal and execute the following command:
+
+    ```
+    curl -sL https://ibm.biz/idt-installer | bash
+    ```
+
+    If you're on a windows machine, open Windows PowerShell by right-clicking and select "Run as Administrator":
+
+    ```
+    Set-ExecutionPolicy Unrestricted; iex(New-Object Net.WebClient).DownloadString('http://ibm.biz/idt-win-installer')
+    ``` 
+
+    This will install the IBM Cloud Developer Tools CLI, the Bluemix CLI, Docker, the Kubernetes CLI, and all other dependencies if they are not already installed.
+
+3. [Create a namespace in the IBM Container Registry](https://console.bluemix.net/docs/services/Registry/index.html#registry_namespace_add), if you do not already have one.
+
+    * Open a terminal or command prompt.
+    * Log into the Bluemix CLI:
+        ```
+        bx login
+        ```
+    * Create a namespace using the `bx cr` command:
+
+        ```
+        bx cr namespace-add <my_namespace>
+        ```
+        
+      This will add a namespace to create your own image repository. Replace `<my_namespace>` with your preferred namespace.
+
+4. Follow all 6 steps in the instructions below to [run locally](#run-locally).
+
+5. Open a terminal and cd into the folder containing the project downloaded in the `run locally` steps.   Within a terminal use the `bx dev enable` command and answer all questions to generate Kubernetes deployment assets.
```

Comment:
Again, in the spirit of a journey, "and answer all questions" really needs to be expanded upon. The questions are effectively non-trivial for a newcomer, and I would expect most of them to be answered incorrectly as a result.